cypress/fixtures/successful_login.json

```diff
@@ -0,0 +1,10 @@
+{
+  "success": true,
```

Comment:
This field does not exist in the login fixture, only the data. Took a look and it seems the fixtures are wrong in most other places as well, but I don't think the other clients are using it any way.
<img width="283" alt="Screen Shot 2020-06-07 at 11 08 53" src="https://user-images.githubusercontent.com/61978848/83964757-72fabe80-a8af-11ea-8a87-59e3103471d4.png">

Reply:
Okey, seems right. Need to change all the fixtures then on all the apps....

Reply:
that's the data from the response...